assets/components/pages/association-homepage/association-homepage.twig

```diff
@@ -54,7 +53,14 @@
 <div class="container">
   {% include '@organisms/contact/contact-banner.twig' %}
```

Comment:
This component was working great on grey background, but with your update, it seems less structured and elegant...

<img width="1310" alt="styleguide" src="https://user-images.githubusercontent.com/2025924/44571688-c6a32300-a781-11e8-8e11-01092cc43808.png">

Reply:
It will surely require a complete review of every page who use that component

Reply:
We checked every page which use this component and add grey background around it when needed